students/thomasvad/lesson2/FizzBuzz.py

```diff
@@ -0,0 +1,10 @@
+def fizzbuzz():
+	for i in range(1,101):
```

Comment:
Over indented - make sure you have your text editor to convert tabs to 4 spaces.  Also, need a space after the comma